internal/service/ec2/vpc_route_table.go

```diff
@@ -811,7 +820,7 @@ func flattenRoute(apiObject *ec2.Route) map[string]interface{} {
 	return tfMap
 }
 
-func flattenRoutes(ctx context.Context, conn *ec2.EC2, apiObjects []*ec2.Route) []interface{} {
+func flattenRoutes(ctx context.Context, d *schema.ResourceData, conn *ec2.EC2, apiObjects []*ec2.Route) []interface{} {
```

Comment:
Nit: Swap `*schema.ResourceData` and `*ec2.EC2` arguments as the most common pattern is `f(ctx, conn, other-args...)`.

Reply:
Yes, absolutely! Careless...